src/features/search/components/VnSearch/VnSearch.tsx

```diff
@@ -0,0 +1,87 @@
+import type { FC } from 'react';
+import React, { useEffect, useState, useCallback, memo } from 'react';
+import { useForm, useWatch } from 'react-hook-form';
+import { useDebounce } from 'src/hooks/useDebounce';
+import { Paginator } from 'src/components/Paginator/Paginator';
+import { Card } from 'src/components/Card/Card';
+import { CardSkeleton } from 'src/components/Card/CardSkeleton';
+import { Form } from 'src/components/Form/Form';
+import { List } from 'src/components/List/List';
+import { PaginationService } from 'src/api/services/paginationService';
+import { DEFAULT_PAGE_SIZE, useVnsQuery } from '../../queries/vns';
+import { VnSearchFormValues, VN_SEARCH_INITIAL_VALUES } from '../VnSearchForm/vnSearchFormValues';
+import { VnSearchForm } from '../VnSearchForm/VnSearchForm';
+
+/** Visual novel overview tab. */
+const VnSearchComponent: FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const methods = useForm({ defaultValues: VN_SEARCH_INITIAL_VALUES });
+  const { control } = methods;
+
+  const formData = useWatch({ control }) as VnSearchFormValues;
```

Comment:
I believe you can use `watch()` from `methods` instead of `useWatch()`. This way it'll have proper typings.

Reply:
`useWatch` has less rerenders